Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -24,6 +30,18 @@ public void setupMyCommands(
     */
     
     @Inject
+    public void setupLauncherCommands(
+            OperatorInterface oi,
+            ShooterSubsystem shooterSubsystem,
+            XPropertyManager propertyManager
+            ) {
+        oi.leftButtons.getifAvailable(6).whenPressed(new IncreaseShooterPowerCommand(shooterSubsystem.getLeftShooter()));
```

Comment:
That doesn't seem like the case currently? There is no other pr's at the moment which conflict with these buttons